README.md

```diff
@@ -1,5 +1,5 @@
-About python-blosc
```

Comment:
Correct. After fixing the name we will need another re-render. Note can do `conda smithy rerender --commit` to commit the changes. Also can tack on `auto` to commit the re-rendering automatically without reviewing the changes.